tests/integration/components/frost-autocomplete-test.js

```diff
@@ -354,8 +372,8 @@ describe(test.label, function () {
               })
             })
           })
-
-          describe('focus into component', function () {
+          // FIXME: Weird async issues on Travis Firefox
```

Comment:
Skipping these tests is not good, but I'm with you - fixing them is not easy! I've ran Firefox 61.0.1 locally on both Mac and Linux with node 8.6.0, and I can't get them to fail. I added a lot of `wait`'s in PR #584 as well, and still no dice.

If we have to skip these for now, you'll need to reset the coverage value in `package.json` so that pr-bumper stops complaining.